controllers/component_build_controller_test.go

```diff
@@ -1816,6 +1828,7 @@ var _ = Describe("Component initial build controller", func() {
 		_ = AfterEach(func() {
 			deleteSecret(pacSecretKey)
 
+			deleteBuildPipelineRunSelector(defaultSelectorKey)
```

Comment:
Why do we need to delete pipeline selector after each of the initial build test?